[2024-09-27T19:29:11.834+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-27T19:29:11.867+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scrape_and_save_share_prices_with_company_data.create_company_table scheduled__2024-04-16T00:00:00+00:00 [queued]>
[2024-09-27T19:29:11.883+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scrape_and_save_share_prices_with_company_data.create_company_table scheduled__2024-04-16T00:00:00+00:00 [queued]>
[2024-09-27T19:29:11.884+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2024-09-27T19:29:11.915+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): create_company_table> on 2024-04-16 00:00:00+00:00
[2024-09-27T19:29:11.930+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=437) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-27T19:29:11.933+0000] {standard_task_runner.py:72} INFO - Started process 440 to run task
[2024-09-27T19:29:11.933+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'scrape_and_save_share_prices_with_company_data', 'create_company_table', 'scheduled__2024-04-16T00:00:00+00:00', '--job-id', '2019', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp80b35sfp']
[2024-09-27T19:29:11.935+0000] {standard_task_runner.py:105} INFO - Job 2019: Subtask create_company_table
[2024-09-27T19:29:12.035+0000] {task_command.py:467} INFO - Running <TaskInstance: scrape_and_save_share_prices_with_company_data.create_company_table scheduled__2024-04-16T00:00:00+00:00 [running]> on host f199b4684a92
[2024-09-27T19:29:12.253+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='scrape_and_save_share_prices_with_company_data' AIRFLOW_CTX_TASK_ID='create_company_table' AIRFLOW_CTX_EXECUTION_DATE='2024-04-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-16T00:00:00+00:00'
[2024-09-27T19:29:12.256+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-09-27T19:29:12.258+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-09-27T19:29:12.259+0000] {logging_mixin.py:190} INFO - Current task name:create_company_table state:running start_date:2024-09-27 19:29:11.868474+00:00
[2024-09-27T19:29:12.260+0000] {logging_mixin.py:190} INFO - Dag name:scrape_and_save_share_prices_with_company_data and current dag run status:running
[2024-09-27T19:29:12.261+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-27T19:29:12.278+0000] {base.py:84} INFO - Retrieving connection 'dse_connection'
[2024-09-27T19:29:33.327+0000] {logging_mixin.py:190} INFO - Error creating company table: connection to server at "172.23.0.3", port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
[2024-09-27T19:29:33.328+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-27T19:29:33.347+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-27T19:29:33.349+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=scrape_and_save_share_prices_with_company_data, task_id=create_company_table, run_id=scheduled__2024-04-16T00:00:00+00:00, execution_date=20240416T000000, start_date=20240927T192911, end_date=20240927T192933
[2024-09-27T19:29:33.367+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-09-27T19:29:33.368+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-09-27T19:29:33.369+0000] {logging_mixin.py:190} INFO - Dag name:scrape_and_save_share_prices_with_company_data queued_at:2024-09-27 19:21:16.731049+00:00
[2024-09-27T19:29:33.370+0000] {logging_mixin.py:190} INFO - Task hostname:f199b4684a92 operator:PythonOperator
[2024-09-27T19:29:33.406+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-27T19:29:33.457+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-27T19:29:33.461+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
